src/components/Home/Footer.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import '../../styles/pages/Home/Footer.css';
+import { Link } from 'react-router-dom';
+import { LogoDiscord, LogoFacebook, LogoInstagram, LogoLinkedin, LogoGithub, LogoYoutube } from 'react-ionicons'
+import pecacm from '../../assets/icons/acm.webp'
+import pec from '../../assets/icons/pec_centenary_logo.jpg'
+function Footer() {
+    return (
+        <footer className="bg-primary text-white">
+            <div className="waves">
+                {/* Your wave divs */}
+            </div>
+
+            <div className="container">
+                <div className="row ">
+                    <div className=" col-md-4 justify-content-center  text-center  p-3">
+                        <h2>Socials</h2>
+                        <ul className="social_icon list-inline d-flex p-3 justify-content-around">
+                            <li>
+                                <Link to="https://www.facebook.com/pecacm/" target="_blank">
+                                    <LogoFacebook className='ion'/>
+                                </Link>
+                            </li>
+                            <li>
+                                <Link to="https://discord.com/invite/59mDGtSyGD" target="_blank">
+                                    <LogoDiscord />
+                                </Link>
+                            </li>
+                            <li>
+                                <Link to="https://www.instagram.com/pecacm/" target="_blank">
+                                    <LogoInstagram />
+                                </Link>
+                            </li>
+                            <li>
+                                <Link to="https://www.linkedin.com/company/pec-acm-student-chapter/" target="_blank">
+                                    <LogoLinkedin />
+                                </Link>
+                            </li>
+                            <li>
+                                <Link to="https://github.com/PEC-CSS/" target="_blank">
+                                    <LogoGithub />
+                                </Link>
+                            </li>
+                            <li>
+                                <Link to="https://www.youtube.com/c/PECACMStudentChapter" target="_blank">
+                                    <LogoYoutube />
+                                </Link>
+                            </li>
```

Comment:
Use `map` to reduce the code size.
You can make array of objects below the `Footer` function.
```js
[
    {
        url: "url_here",
        icon: <IconComponent />    
    },
    ....
]
```
Each element will look something like this.